### PR TITLE
Add CPDB Page, setup.py, and update python in dockerfile

### PR DESCRIPTION
**Pull request by mbh329**
**Add CPDB Page**

Addresses issue #46 

Nothing new happening here. These changes were all pair coded by the team earlier this week in which we added the cpdb page:

- Create folder structure similar to `facdb` and `pluto` where we have two .py files (helpers.py and cpdb.py) 
- Get the data from DO edm-private space
- View the `Managing Agency Summary Stats` and `Sponsoring Agency Summary Stats` tables in the app 

**Update setup.py**

Addresses issue #57 

The setup.py file is outdated after our latest changes to the app. When I was testing the app locally, I was not able to view the latest changes so I added the necessary modules into the setup.py which should fix any issues. You will need to rebuild the dev container to test these changes. Also make sure you have the latest secrets in your .env. Eventually we might just want to use poetry but in the meantime this is a necessary change to keep the qaqc development moving 

**Update python version in dev container**

Addresses issue #58 
I updated the version of python in the dev container from 3.6 to 3.9. I noticed when rebuilding my container that 3.6 is no longer supported by boto3 so thought this was a necessary updgrade -  it is time we updated to a newer version of python anyway because python 3.6 is a few years old at this point. This should be tested further by each engineer but I don't think (???) it should cause any issues.

### CODE REVIEW

**Comment by Oysters1874** [comment]
Yea, I think we should update the python version to 3.9, otherwise it has caused several problem when installing packages. To keep things moving, we can make those changes at this moment and later maybe move to poetry.

File: src/cpdb/helpers.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import os
+import pandas as pd
+import boto3
+from dotenv import load_dotenv
+
+load_dotenv()
+
+def get_data(branch) -> dict:
+    rv = {}
+    # url = f"https://edm-private.nyc3.digitaloceanspaces.com/db-cpdb/{branch}/latest/output/analysis"
```
Comment (abrieff) [inline src/cpdb/helpers.py:12]: probably want to remove all these comments

**Review by abrieff — APPROVED** [review]
lgtm

File: src/cpdb/helpers.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    for table in ['magency', 'sagency']:
+        obj = client.get_object(
+            Bucket="edm-private",
+            Key=f"db-cpdb/{branch}/latest/output/analysis/cpdb_summarystats_{table}.csv",
+        )
+        s = str(obj["Body"].read(), "utf8")
+        data = StringIO(s)
+        df = pd.read_csv(data, encoding='utf8')
+        rv[table] = df
+    # rv['magency'] = pd.read_csv(f"{url}/cpdb_summarystats_magency.csv")
```
Comment (td928) [inline src/cpdb/helpers.py:29]: also those comments as well @mbh329

**Pull request merged**